App/Properties.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace App
+{
+    /// <summary>
+    /// Class used to store and load properties. It is used to
+    /// </summary>
+    /// <author>Nick Rimmer (https://stackoverflow.com/questions/485659/can-net-load-and-parse-a-properties-file-equivalent-to-java-properties-class#answer-7696370</author>
```

Comment:
```suggestion
    /// <author> We used the code published by Nick Rimmer 
    Link : (https://stackoverflow.com/questions/485659/can-net-load-and-parse-a-properties-file-equivalent-to-java-properties-class#answer-7696370
    </author>
```

Otherwise this looks like we just copy pasted the code without thinking about it.
Also will Monmarché not like us using code we found online ?

Reply:
Then we can Rewrite it but that would almost be the same thing, as you want